akka-bbb-apps/project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.2.8
+sbt.version=1.5.5
```

Comment:
Note to @antobinary -- we need to reflect this change in https://gitlab.senfcall.de/senfcall-public/docker-bbb-build/-/blob/main/Dockerfile#L135-137 in a new tag for 2.5